app/build.gradle

```diff
@@ -77,6 +78,7 @@ android {
             buildConfigField "boolean", "ENGAGE_GOOGLE_PLAY_RATING", "true"
             buildConfigField "String", "TRACE_DROID_EMAIL_ADDRESS", '"tobias.preuss+camp2019@googlemail.com"'
             buildConfigField "String", "SCHEDULE_FEEDBACK_URL", '"https://frab.cccv.de/en/camp2019/public/events/%s/feedback/new"'
+            buildConfigField "boolean", "ENABLE_CHAOSFLIX_EXPORT", "true"
```

Comment:
Please remove this redundant setting from the `cccamp2019` flavor since it is enabled by default.

Reply:
Resolved in 5147a5af7b566465f39d8b1c4b3c483cdf9416f9. Thank you.